charts/kyverno-aws-adapter/README.md.gotmpl

```diff
@@ -15,11 +15,18 @@ You’ll need an [EKS](https://aws.amazon.com/eks/) cluster to run against.
 ### Running on the EKS cluster
 
 1. Make sure that you have configured an [IAM role for the service account](#IAM-Role-for-Service-Account) `kyverno-aws-adapter-sa` in your desired namespace (configured in `values.yaml`) and specified the role's ARN in the `roleArn` field inside `values.yaml` file.
-2. Install the Helm chart after making any necessary changes to `charts/kyverno-aws-adapter/values.yaml`
+2. Add the Kyverno AWS Adapter Helm repository.
+
+```console
+helm repo add kyverno-aws-adapter https://nirmata.github.io/kyverno-aws-adapter/
+helm repo update kyverno-aws-adapter
+```
+
+3. Install the Helm chart providing parameters as described in the [Values](#values) section.
```

Comment:
```
   helm install kyverno-aws-adapter charts/kyverno-aws-adapter
```

This is incorrect now, should reference the repo added above.

Reply:
Fixed.

Reply:
Nope

Reply:
Changed now.